_data/social.yml

```diff
@@ -249,7 +249,7 @@ websites:
       twitter: vanilla
 
     - name: Viadeo
-      url: http://www.viadeo.com
+      url: https://www.viadeo.com
```

Comment:
This site redirects to a HTTP version for me.
Is it just me or is anyone else getting this as well?

Reply:
I can see that as well. I think there's not really a point in linking the https when it redirects back to http anyway, right?

Reply:
In my case, as I reside in Norway, it redirects to https://no.viadeo.com/en/ with what seems to be fully functional HTTPS.